example/transform/openai_jupyter_notebook_QA.ipynb

```diff
@@ -355,7 +344,7 @@
     "    guided_prompt_template=guided_prompt,\n",
     "    model_config=OpenAIModelConfig(response_format={\"type\": \"json_object\"}),\n",
     ")\n",
-    "client = TransformClient(config)"
+    "client = Client(config)"
```

Comment:
same here

Reply:
Also, in `huggingface_model.ipynb`, the config import is 
`from uniflow.flow.config import TransformHuggingFaceConfig, HuggingfaceModelConfig`
Do we need to seperate it since it is used differently from other files?

Reply:
Do you mean separate as below?
```
from uniflow.flow.config import TransformHuggingFaceConfig
from uniflow.op.model.model_config import HuggingfaceModelConfig
```

Reply:
Yes, it makes sense to be consistent.